style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  padding-left: 30px;
+  padding-right: 30px;
+  font-family: 'Poppins', sans-serif;
+  background: url("images/background-image.svg");
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#menu-icon {
+  width: 20px;
+  height: 20px;
+  background-color: #6070ff;
+}
+
+#logo {
+  color: #6070ff;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#primary-text {
+  display: flex;
+  padding-right: 60px;
+  font-weight: bold;
+  font-size: 40px;
+  padding-top: 70px;
+  line-height: 1.3;
+}
+
+#secondary-text {
+  display: flex;
+  padding-top: 10px;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+#connect {
+  display: flex;
+  color: #6070ff;
+  font-size: 16px;
+  padding-top: 15px;
+}
+
+#social {
+  padding-top: 10px;
+  padding-right: 200px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
kindly the cursor must be pointer. because so far in the project you will assign each icon to your public social media link.